Index: day01.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- day01.py	(date 1664080240106)
+++ day01.py	(date 1664080240106)
@@ -0,0 +1,30 @@
+# 04,11,13,19,22,33   01,07,13,17,18,31
+c = []
+
+
+def LiangShuXiangJIA(aa):
+    a = aa
+    c = []
+    for i in range(0, len(a) - 1):
+        b = int(a[i]) + int(a[i + 1])
+        c.append(b)
+        # print(b)
+    return c
+
+
+def LiangShuXiangJian(aaa):
+    a = aaa
+    c = []
+    for i in range(len(a) - 1, 0, -1):
+        b = int(a[i]) - int(a[i - 1])
+        c.append(b)
+        # print(b)
+    return c
+
+
+if __name__ == '__main__':
+    a = [4, 11, 13, 19, 22, 33]
+    zz = LiangShuXiangJian(a)
+    zz1 = LiangShuXiangJIA(a)
+    print('两数相加', zz1, '两输相减', zz, )
+    print('原数据：04,11,13,19,22,33')
Index: 练习.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- 练习.py	(date 1664080240116)
+++ 练习.py	(date 1664080240116)
@@ -0,0 +1,39 @@
+import os
+
+b = "子"
+a = '{}曰：学而实习之，不亦说乎；{}又曰：友朋自远方来，不亦君子呼！{}'.format(b, b, b)
+
+print(a)
+print('%s曰：学而实习之，不亦说乎；%s又曰：友朋自远方来，不亦君子呼！%s' % (b, b, b))
+
+print()
+
+# a=os.system('dir')
+# print(a)
+# b=os.system('mkdir zzz')
+# print(b)
+#
+# a=os.system('dir')
+# print(a)
+import random
+
+n = 2
+while n > 0:
+    z = int(input("输入一个数据： "))
+    b = random.randint(1, 50)
+    print(b)
+    i = 0
+    for i in range(i, z):
+
+        if i == 0:
+            continue
+        else:
+            print('第{}次'.format(i))
+        if i == z - 1:
+            break
+        else:
+            pass
+
+            j = 0
+            for j in range(1, i + 1):
+                print('*', end='|')  # end=" "
